torch/nn/modules/sparse.py

```diff
@@ -50,6 +50,24 @@ class Embedding(Module):
         output. The gradient for this vector from :class:`~torch.nn.Embedding`
         is always zero.
 
+    .. note::
+        When :attr:`max_norm` is not ``None``, :class:`Embedding`'s forward method will modify the
+        :attr:`weight` Tensor in-place, rather than setting ``Embedding.weight`` to a new Tensor.
```

Comment:
I think we can kill the part of this sentence after the comma? 

Also "Tensor" -> "tensor" in this sentence.